de.dlr.sc.virsat.product/virsat_core.product

```diff
@@ -142,7 +143,6 @@ Germany
       <feature id="org.eclipse.rmf.reqif10.feature" installMode="root"/>
       <feature id="org.eclipse.rmf.reqif10.pror.feature" installMode="root"/>
       <feature id="org.eclipse.rmf.reqif10.thirdparty.feature" installMode="root"/>
-      <feature id="de.dlr.sc.virsat.model.extension.thermal.feature" installMode="root"/>

```

Comment:
Removal of thermal doesn't look right

Reply:
I removed it because it is included two times in the product file (line 145 and line 90)